package.json

```diff
@@ -35,7 +36,7 @@
         "babel-loader": "^8.0.6",
         "babel-plugin-transform-require-ignore": "^0.1.1",
         "box-annotations": "^2.3.0",
-        "box-ui-elements": "^13.1.0-beta.62",
+        "box-ui-elements": "^15.0.0-beta.49",
```

Comment:
This is a big jump. Do all our BUIE usages still work as expected?

Reply:
Will do some regression testing

Reply:
Looks good, tested CSV viewer, archive viewer, annotations